erp-server/src/main/java/com/ttt/erp/service/AwardPdfService.java

```diff
@@ -78,8 +81,11 @@ private static ByteArrayInputStream generateAwardPdf(Award award) throws IOExcep
         document.add(new Paragraph(userAccount.getUsername()));
 
         if (userAccount.getSignature() != null) {
-            String signatureSrc = "signatures/" + userAccount.getSignature();
-            Image signature = new Image(ImageDataFactory.create(signatureSrc)).scaleAbsolute(100, 100);
+            Image signature = new Image(
+                ImageDataFactory.create(
+                    signatureService.getSignatureBytes(userAccount.getSignature())
+                )
+            ).scaleAbsolute(100, 100);
```

Comment:
This now gets the byte array via SignatureService, which returns the byte array from the file on AWS S3. For any users having signatures not in S3, this will break right now. But we'll be cleaning up all of the users anyways prior to deploy.